Public/Profile.html

```diff
@@ -122,12 +136,25 @@
                 content += "...";
             }
 
-            const body = `<div class="article" style="cursor: pointer;" onclick="navigateTo('./Article.html?id=${element.post_id}')"><h2>${element.title}</h2><h4>${element.username}</h4><p>${content}</p></div>`
+            const body = `<div class="article" style="cursor: pointer;"><div class="articleBody" onclick="navigateTo('./Article.html?id=${element.post_id}')"><h2>${element.title}</h2><h4>${element.username}</h4><p>${content}</p></div> <i class="bi bi-x-lg deleteButton" style="font-size: 2rem; cursor: pointer;" onclick="deletePost(${element.post_id})"></i></div>`
             let row = table.insertRow();
             row.innerHTML = body;
         });
     }
 
+    function deletePost(postID) {
+        //TODO: Implement delete fetch call
+        fetch("http://localhost:8000/post", {
```

Comment:
I do not believe this route exists yet. The only DELETE route I see is for "/", and is intended to delete comments.

Reply:
I misunderstood how the route seperation worked. This is properly implemented.